src/components/Button.tsx

```diff
@@ -562,7 +564,7 @@ export function ButtonIcon({
         size={
           iconSize ?? (size === 'large' ? 'md' : size === 'tiny' ? 'xs' : 'sm')
         }
-        style={[{color: textStyles.color, pointerEvents: 'none'}]}
+        style={[{color: color ?? textStyles.color, pointerEvents: 'none'}]}
```

Comment:
Yeah could pry drop this, would rather not encourage its use. Happy to discuss though!

Reply:
Yeah np, I don't think I ended up needing it